atc/worker/placement.go

```diff
@@ -69,18 +74,30 @@ func NewChainPlacementStrategy(opts ContainerPlacementStrategyOptions) (*ChainPl
 		case "fewest-build-containers":
 			cps.nodes = append(cps.nodes, newFewestBuildContainersStrategy(strategy))
 
+		case "fewest-active-tasks":
+			cps.nodes = append(cps.nodes, newFewestActiveTasksStrategy(strategy))
+
 		case "limit-active-tasks":
 			if opts.MaxActiveTasksPerWorker < 0 {
 				return nil, errors.New("max-active-tasks-per-worker must be greater or equal than 0")
 			}
 			cps.nodes = append(cps.nodes, newLimitActiveTasksStrategy(strategy, opts.MaxActiveTasksPerWorker))
 
+			// For backwards compatibility, "limit-active-tasks" implies "fewest-active-tasks"
+			cps.nodes = append(cps.nodes, newFewestActiveTasksStrategy(strategy))
+
+		case "fewest-active-containers":
```

Comment:
Would it be possible to revert most of the changes in this commit, and instead go with the partitioning approach in https://github.com/concourse/concourse/pull/6635#discussion_r600459319? The changes make sense, but I'd like to keep the scope of this PR limited. We'll need to think about the placement strategies a little more, first - e.g. `fewest-active-containers` is pretty much just `fewest-build-containers` now